shared/structures/src/webshops/Checkout.ts

```diff
@@ -250,18 +250,19 @@ export class Checkout extends AutoEncoder {
         // Check maximum
         if (timeSlot.remainingPersons !== null && this.cart.persons - this.reservedPersons > timeSlot.remainingPersons) {
             const remaingPersons = timeSlot.remainingPersons
+            const availableTimeslots = this.checkoutMethod.timeSlots.timeSlots.length
             if (remaingPersons === 0) {
                 throw new SimpleError({
                     code: "timeslot_full",
                     message: "Timeslot has reached maximum orders",
-                    human: "Het gekozen tijdstip is helaas volzet. Kies een ander tijdstip indien mogelijk.",
+                    human: (availableTimeslots !=1 ? "Het gekozen tijdstip is helaas volzet. Kies een ander tijdstip indien mogelijk." : "Het evenement is helaas volzet. We aanvaarden geen verdere bestellingen."),
```

Comment:
Een paar lijnen hierboven moet je het best ook nog wijzigen (bij maximale bestellingen) in plaats van maximale personen.

Reply:
Toegevoegd!

Ik was nog aan het proberen om ergens in tekst uit te leggen hoe het werkt en wanneer welke info getoond wordt. Maar bij nader inzien denk ik dat alles logisch is en dat extra uitleg niet bijdraagt aan de helderheid.